docs/book/v3/migration/v2-to-v3.md

```diff
@@ -48,6 +48,36 @@ The impact of the removal of these aliases will not affect you if you use a FQCN
 
 ### Changes to Individual Filters
 
+#### `CamelCaseToDash`
```

Comment:
Assuming #10 is fixed, we might need an extra paragraph here to describe the behaviour change too